flower/views/broker.py

```diff
@@ -24,8 +24,12 @@ def get(self):
             http_api = app.options.broker_api
 
         broker = Broker(app.capp.connection().as_uri(include_password=True),
-                        http_api=http_api)
+                        http_api=http_api, celery_conf=self.capp.conf)
         queue_names = ControlHandler.get_active_queue_names()
```

Comment:
I think it would be better to pass only BROKER_TRANSPORT_OPTIONS. Broker should get only broker specific options.